### PR TITLE
Add preliminary host-guest analysis for CB8

### PR DESCRIPTION
**Pull request by amezcum1**
- Adds preliminary analysis and results for the CB8 challenge
- Updates to submission files and the user map

### CODE REVIEW

File: host_guest/Analysis/Submissions/CB8/CB8-sampl8-submission-LiGaMD.txt [hunk 1]
```diff
@@ -1,3 +1,110 @@
+<<<<<<< HEAD
```
Comment (davidlmobley) [inline host_guest/Analysis/Submissions/CB8/CB8-sampl8-submission-LiGaMD.txt:1]: You've got some `<<<<` and `>>>>` marks from a git merge conflict it looks like; probably need to remove them.

File: host_guest/Analysis/Submissions/CB8/CB8_submission_ARNOLD.txt [hunk 2]
```diff
@@ -34,8 +34,8 @@ CB8-G4, -16.0, 0.2, 1.0,,,
 CB8-G5, -13.8, 0.2, 1.0,,,
 CB8-G6, -11.6, 0.2, 1.0,,,
 CB8-G7, -14.3, 0.2, 1.0,,,
-CB8-G8, -9.6, 0.2, 1.0,,,
-CB8-G9, -9.4, 0.2, 1.0,,,
+#CB8-G8, -9.6, 0.2, 1.0,,,
```
Comment (davidlmobley) [inline host_guest/Analysis/Submissions/CB8/CB8_submission_ARNOLD.txt:37]: Why are these commented out?

**Review by davidlmobley — APPROVED** [review]
In general looks good! I added a couple small comments on things to clean up, but then you're good to merge.

One small thing I noticed to posssibly address in a lter PR is that for some reason, the mean error distribution violin plots seem to be out of order.

File: host_guest/Analysis/Submissions/CB8/CB8_submission_ARNOLD.txt (hunk 2 above)
Comment (amezcum1) [reply host_guest/Analysis/Submissions/CB8/CB8_submission_ARNOLD.txt:37]: These systems are not part of challenge, commented for initial analysis. Missed changing back. Analysis script should be able to handle/ignore them

**Pull request merged**